util/pull_dataset.py

```diff
@@ -3,7 +3,7 @@
 import os
 
 import click
-from jinacld_tools.aws.services.s3 import S3
+from jinacld_tools.aws.services.s3 import S3Bucket
```

Comment:
Won't this fail, since it's expecting `S3` below, at https://github.com/jina-ai/examples/blob/chore-update-cloud-helper/util/pull_dataset.py#L43

Reply:
Oh shoot, thank Cristian.

Reply:
257e54f6e340c6b9f4eb1d4d530bfa91b0e76725